debug/index.html

```diff
@@ -239,6 +239,7 @@ <h1 class="title">Transmux Analyzer</h1>
       codecsArray = codecs.value.split(',');
 
       mediaSource.addEventListener('sourceopen', function () {
+        mediaSource.duration = 0;
```

Comment:
... keep forgetting this is the debug page. Less important but it still seems like it could mislead *us* at some point.